stories/topics/ref-aws-additional-configs-update-admin-password.adoc

```diff
@@ -13,5 +13,5 @@ Follow these steps to change the Administrator password for both {ControllerName
 . Replace the running {PlatformNameShort} EC2 instances to use the new Administrator password.
 .. Navigate to the AWS EC2 Instances console.
 .. Identify and Terminate one {ControllerName} instance and one {HubName} instance for the CloudFormation Stack.
```

Comment:
It seems that you are only removing one of the two controllers. Wouldn't this cause one controller with old pwd and one with new?

Reply:
Only have to update the password in the database, so one controller is enough to perform the update.